casts/splitNavigator/src/SplitNavigator/SplitBottomTabBar.tsx

```diff
@@ -0,0 +1,305 @@
+/* eslint-disable react/no-unused-prop-types */
+import * as React from 'react';
+import { View, ImageSourcePropType, StyleProp, ViewStyle, StyleSheet } from 'react-native';
+import {
+    GestureEvent,
+    NativeViewGestureHandlerPayload,
+    NativeViewGestureHandlerProps,
+    RawButton as GHRawButton,
+    RawButtonProps,
+} from 'react-native-gesture-handler';
+import ReAnimated, {
+    runOnJS,
+    useAnimatedGestureHandler,
+    useAnimatedStyle,
+    useSharedValue,
+    withSpring,
+} from 'react-native-reanimated';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { UIImage } from '@tonlabs/uikit.media';
+import { useTheme, ColorVariants } from '@tonlabs/uikit.themes';
+import { hapticSelection } from '@tonlabs/uikit.controls';
+import { ShadowView } from './ShadowView';
+
+type SplitScreenTabBarAnimatedIconComponent = React.ComponentType<{
```

Comment:
Maybe export this type and reuse it for MainAnimatedIcon? (and let others make such icons knowing the desired interface of them)